### ISSUE TITLE
"import traits.api" raises ImportError for version 4.4

### ISSUE THREAD

**Issue by lesteve** [ContinuumIO/anaconda-issues#767]
To reproduce:

```bash
conda create -n del python traits=4.4 -y
source activate del
python -c 'import traits.api'
```

Error:
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/lesteve/miniconda3/envs/del/lib/python2.7/site-packages/traits/api.py", line 26, in <module>
    from .trait_base import Uninitialized, Undefined, Missing, Self, python_version
  File "/home/lesteve/miniconda3/envs/del/lib/python2.7/site-packages/traits/trait_base.py", line 34, in <module>
    from . import _py2to3
ImportError: cannot import name _py2to3
```

The only reason I care about traits=4.4 and I can not use the latest version available in conda (i.e. 4.5) is because 4.4 is the latest version that is compatible with mayavi, at least according to conda:

```
conda create -n del traits=4.5 mayavi
```

```
The following specifications were found to be in conflict:
  - mayavi -> apptools 4.2.0|4.2.1
  - mayavi -> traitsui 4.3.0|4.4.0
  - traits 4.5*
```

**Comment by lesteve** [comment]
Seems like there is a similar issue with traitsui=4.4. To reproduce it:

```bash
conda create -n del python traitsui=4.4 -y
source activate del
python -c 'import traitsui'
```

Error:
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/lesteve/miniconda3/envs/del/lib/python2.7/site-packages/traitsui/__init__.py", line 20, in <module>
    from ._version import full_version as __version__
ImportError: No module named _version
```

**Comment by lesteve** [comment]
Hmm somehow I can not reproduce the problem if I download a fresh version of the miniconda installer.

It looks like I ended up with a mix of 4.4 and 4.5 versions in ~/miniconda3/pkgs/traits-4.4.0-py27_0 maybe by mixing conda and pip. I ended up manually removing the traits and traitsui folders from ~/miniconda3/pkgs with `rm -rf` and conda removing + conda installing again.

Not sure whether there was an easier way to get back to a sane environment ...
